src/Mod/Path/PathScripts/PathSurface.py

```diff
@@ -1046,15 +1048,23 @@ def _wireToFlatCrsSctn(self, mBB, wire):
         cont = True
         save = False
         extFwd = (mBB.ZLength * 2.1)
+        zmin = mBB.ZMin - 1.0
         zmax = mBB.ZMin + extFwd
-        stpDwn = (zmax - mBB.ZMin) / 4
-        dep_par = PathUtils.depth_params(zmax + 5, zmax + 3, zmax, stpDwn, 0.0, mBB.ZMin)
+        stpDwn = (zmax - zmin) / 4

```

Comment:
Although I think Path is no longer compatible with Python 2, I still recommend to make the divisions with floats, in case somebody else decides to use Python 2 still.
```
stpDwn = (zmax - zmin) / 4.0
```
Divisions in Python 2 always require you to explicitly use a float to get the correct floating point result.

Reply:
Sure thing. I catch this sometimes, but I'm not consistent.  I'm on a mobile now, so I'll make a mental note to take care of that. Thanks.